modules/widgets/facebook-likebox.php

```diff
@@ -42,6 +37,8 @@ function widget( $args, $instance ) {
 
 		$like_args = $this->normalize_facebook_args( $instance['like_args'] );
 
+		wp_enqueue_style( 'jetpack_facebook_likebox', plugins_url( 'facebook-likebox/style.css', __FILE__ ) );
```

Comment:
This feels kinda silly for a three line css file.

Will it work properly if we add in a `wp_style_add_data( 'jetpack_facebook_likebox', 'jetpack-inline', true );` to it to trigger `Jetpack::maybe_inline_style()` ?

https://github.com/Automattic/jetpack/blob/990783ba568ed7935a47174fdb9756943f57a923/class.jetpack.php#L5320-L5382